src/Core/Checker.cpp

```diff
@@ -315,7 +313,7 @@ void Checker::check(int index, const QString &input, const QString &output, cons
             Util::saveFile(expectedPath, expected, tr("Checker"), false, log))
         {
             // if files are successfully saved, run the checker
-            auto tmp = new Runner(index);
+            auto *tmp = new Runner(index);
```

Comment:
Qualified `auto`, so that `decltyp(auto)` returns only type and not pointer to type. It is modern convention.